docs/NNNN-schema-registry.md

```diff
@@ -0,0 +1,195 @@
+Extension Name: NNNN-schema-registry
+Authors: P. Cornwell, D. Granville
+Minimum OCFL Version: 1.0
+OCFL Community Extensions Version: 1.0
+Extension Version: 0.1
+Obsoletes: n/a
+Obsoleted by: n/a
+
+## Overview
+An OCFL object will typically contain metadata serialised in JSON or XML files. These must currently conform to one or more schemata referenced as external JSONSchema/XSD/DTD files.
```

Comment:
Not true, there is no requirement for metadata inside objects to conform to anything although it is clearly good practice. This should be edited to make that clear

Reply:
I'm not sure where that language crept in, probably trying to avoid another use of 'typically'. Agree and will change it - "nominally" hasn't featured yet...